usaspending_api/api_docs/api_documentation/advanced_award_search/transaction_sum.md

```diff
@@ -0,0 +1,41 @@
+## [Transaction Sum](#transaction-sum) - Deprecated
```

Comment:
This route is not used and does not exist on the API side either.

REMOVE

Reply:
Gone